lib/arrays.md

```diff
@@ -5,44 +5,58 @@ instructor_notes: Feel free to re-organize the headings (or add/remove headings)
 
 # What is an Array?
 
-Replace me with your response...
+An array is used to organize information into an ordered list. 
+The list can be made up of strings, integers, floats, booleans, or any other data type. 
 
 # What are some examples of information that would be Arrays as opposed to some other data type?
 
-Replace me with your response...
+In Ruby, an array is denoted by square brackets [ ]. Inside the brackets you can create a list of elements separated by commas. 
+The brackets define that it would an array.
 
 # What is one way, using Ruby, to retrieve the 6th element in an Array? How about the 8th element? What happens if you try to retrieve the value of the _9999th_ element (or some element that doesn't exist in the array)?
 
-Replace me with your response...
+Each element in an array can be accessed via an index. 
+For example [ 1, 2, 3, 4, 5][0] would get us the '1' from the list because the values start at '0'.
+if you try to retrieve the 9999th number it would come back as 'nil' or nonexistant.
 
 # The previous question asks about finding, for example, the 6th element in an Array. Is it possible to find the **-6th** (Notice the negative symbol!) element in an Array? What does that even mean?
 
-Replace me with your response...
+it is possible to retrieve the -6th element. If I did [ 1, 2, 3, 4, 5, 6, 7, 8, 9][-6] it would give me 4. 
+It starts at the end of my index and counts backwards 6.
 
 # How would you perform an operation on every element inside an Array?
 
-Replace me with your response...
+you would use something similar to Array.[](...) [or] Array[...] [or] [...] depending on what it is you wanted to accomplish. Like Array.index or Array.length.
```

Comment:
That's not quite what I have in mind (and I'm not sure I understand you completely either).

I'd like you to look up _loops_ and write out some code that would loop through all of the items in the `friends` array and execute the `send_letter` function on each friend. See if this guidance helps, but feel free to follow up if it's not clear.